api/service/networkinfo/service.go

```diff
@@ -115,7 +115,18 @@ func (s *Service) DoService() {
 				return
 			}
 			if peer.ID != s.Host.GetP2PHost().ID() && len(peer.ID) > 0 {
-				utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs)
+				utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs, "my ID", s.Host.GetP2PHost().ID())
+				var wg sync.WaitGroup
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					if err := s.Host.GetP2PHost().Connect(s.ctx, peer); err != nil {
+						utils.GetLogInstance().Warn("can't connect to peer node", "error", err)
+					} else {
+						utils.GetLogInstance().Info("connected to peer node", "peer", peer)
+					}
+				}()
+				wg.Wait()
 				ip := "127.0.0.1"
```

Comment:
this does not belong to this PR but i want to discuss.

1/ should be set as constnat
2/ what's the logic behind that. why do we only add non-127.0.0.1 ip to peerChan?